packages/web/src/lib/logger.ts

```diff
@@ -0,0 +1,52 @@
+declare const debug: debug.Debug & { debug: debug.Debug; default: debug.Debug };
+// eslint-disable-next-line @typescript-eslint/no-namespace
+declare namespace debug {
+	interface Debug {
+		(namespace: string): Debugger;
+		coerce: (val: any) => any;
+		disable: () => string;
+		enable: (namespaces: string) => void;
+		enabled: (namespaces: string) => boolean;
+		formatArgs: (this: Debugger, args: any[]) => void;
+		log: (...args: any[]) => any;
+		selectColor: (namespace: string) => string | number;
+		humanize: typeof import("ms");
+
+		names: RegExp[];
+		skips: RegExp[];
+
+		formatters: Formatters;
+	}
+
+	type IDebug = Debug;
+
+	interface Formatters {
+		[formatter: string]: (v: any) => string;
+	}
+
+	type IDebugger = Debugger;
+
+	interface Debugger {
+		(formatter: any, ...args: any[]): void;
+
+		color: string;
+		diff: number;
+		enabled: boolean;
+		log: (...args: any[]) => any;
+		namespace: string;
+		destroy: () => boolean;
+		extend: (namespace: string, delimiter?: string) => Debugger;
+	}
+}
+
+const log = debug("algob/web:log");
+
+const error = debug("algob/web:error");
+error.log = console.error.bind(console);
```

Comment:
why do we need this?

Reply:
It shows logs in different colors, error is shown in red, log in yellow

Reply:
yes for `console.error(msg) => error(msg) and for console.log(msg) => log(msg)` can be used.